packages/datagateway-dataview/src/page/pageContainer.component.tsx

```diff
@@ -476,49 +478,58 @@ class PageContainer extends React.Component<
 
   public render(): React.ReactElement {
     return (
-      <Paper square elevation={0} style={{ backgroundColor: 'inherit' }}>
-        <NavBar
-          entityCount={this.props.entityCount}
-          cartItems={this.props.cartItems}
-          navigateToSearch={this.props.navigateToSearch}
-          navigateToDownload={this.props.navigateToDownload}
-        />
+      <SwitchRouting location={this.props.location}>
+        {/* Load the homepage */}
+        <Route exact path={paths.homepage} component={TranslatedHomePage} />
+        <Route
+          render={() => (
```

Comment:
Thanks to use of the `render` function, the original functionality of pageContainer remains without being remounted all the time. This also removes the need for an entirely new component dedicated just to pageContainer, although if it helps readability and maintenance down the line, this can also be done.